pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/entities/Gym.java

```diff
@@ -87,6 +88,12 @@ public void setGymLeader(Trainer gymLeader) {
         this.gymLeader = gymLeader;
     }
 
+    public Set<Badge> getBadges() { return Collections.unmodifiableSet(this.badges); }
+
+    public void addBadge(Badge badge) { this.badges.add(badge); }
+
+    public void removeBadge(Badge badge) { this.badges.remove(badge); }
```

Comment:
Please on separate line as the other getters and setters, just a detail but makes methods in class more readable.